docs/as-a-resource/from-data-to-array.md

```diff
@@ -3,21 +3,6 @@ title: From data to array
 weight: 1
 ---
 
-A data object can automatically be transformed into an array as such:
-
-```php
-SongData::from(Song::first())->toArray();
-```
-
-Which will output the following array:
-
-```php
-[
-    'name' => 'Never gonna give you up',
-    'artist' => 'Rick Astley'
-]
-```
-
```

Comment:
I think this is removed due to being repeated on line 30 but let's keep this introduction.

Reply:
@rubenvanassche Yep, exattly. As it will be repeated literraly right after it. So you can count the next one as the intro too 👀
Just give the last word on reverting or keeping this change and I'll update it accordingly. 💪 
![image](https://github.com/spatie/laravel-data/assets/24431504/732fe26c-c410-4fe5-a037-18e8ebb01855)